src/deeplinks.ts

```diff
@@ -20,5 +20,5 @@ export const initDeeplinks = (app: Electron.App) => {
 
 
 export const deeplinkHandler = () => {
-   dialog.showMessageBoxSync({title: "Rookout Desktop App", message: "Rookout Desktop App is now running"});
+   dialog.showMessageBoxSync({title: "Dynatrace Desktop App", message: "Dynatrace Desktop App is now running"});
```

Comment:
@NoaAylonDyn @ron-rookout Shouldn't the protocol be changed too? It is still "rookout"

Reply:
> @NoaAylonDyn @ron-rookout Shouldn't the protocol be changed too? It is still "rookout"

@dan505512 you too i guess lol

Reply:
Also under "protocols" in package.json